itests/hive-unit/src/test/java/org/apache/hadoop/hive/ql/parse/TestReplicationScenariosUsingSnapshots.java

```diff
@@ -79,11 +80,11 @@ public static void classLevelSetup() throws Exception {
     overrides.put(HiveConf.ConfVars.REPL_INCLUDE_EXTERNAL_TABLES.varname, "true");
     overrides.put(HiveConf.ConfVars.HIVE_DISTCP_DOAS_USER.varname,
         UserGroupInformation.getCurrentUser().getUserName());
-    overrides.put(HiveConf.ConfVars.REPL_RUN_DATA_COPY_TASKS_ON_TARGET.varname, "false");
+    overrides.put(HiveConf.ConfVars.REPL_RUN_DATA_COPY_TASKS_ON_TARGET.varname, "true");
     overrides.put(HiveConf.ConfVars.REPL_EXTERNAL_WAREHOUSE_SINGLE_COPY_TASK.varname, "true");
     overrides.put(REPL_SNAPSHOT_DIFF_FOR_EXTERNAL_TABLE_COPY.varname, "true");
 
-    internalBeforeClassSetup(overrides, TestReplicationScenarios.class);
+    internalBeforeClassSetupExclusiveReplica(overrides, overrides, TestReplicationScenarios.class);
```

Comment:
why do we need this? all the test should work as is without bothering existing tests.

Reply:
The tests are modified to use different filesystems for src and tgt. A few wrongFS related errors were fixed and this required testing with different fs for src and tgt.